UI/Web/src/app/series-detail/series-detail.component.html

```diff
@@ -52,7 +52,7 @@ <h4 class="offcanvas-title" id="offcanvas-basic-title">Page Settings</h4>
 </app-side-nav-companion-bar>
 
 
-<div (scroll)="onScroll()" class="main-container container-fluid pt-2" *ngIf="series !== undefined" #scrollingBlock>
+<div (scroll)="onScroll()" [style.height]="getHeight()" class="main-container container-fluid pt-2" *ngIf="series !== undefined" #scrollingBlock>
```

Comment:
Main container has the #scrollingBlock, which means we can use a ViewChild to get this on load and keep a reference to it. 

Also, you should use [ngStyle]="{'height': getHeight()}"